neural_network/simple_neural_network.py

```diff
@@ -0,0 +1,53 @@
+"""
+Forward propagation explanation:
+https://towardsdatascience.com/forward-propagation-in-neural-networks-simplified-math-and-code-version-bbcfef6f9250
+"""
+
+import random as ran
+import math
+
+
+# Sigmoid
+def sigmoid_function(value: float, deriv: bool = False) -> float:
+    """Return the sigmoid function of a float.
+
+    >>> sigmoid_function(3.5)
+    0.9706877692486436
+    >>> sigmoid_function(3.5, True)
+    -8.75
+    """
+    if deriv:
+        return value * (1 - value)
```

Comment:
```suggestion
           return value * (1 - value)
```

Reply:
I don't think this is right, are you sure of this change?

Reply:
> I don't think this is right, are you sure of this change?

It seems fixed now, perhaps it was a bug? On my screen, it seemed like there was only 1 space after the `if`